docs/api.yml

```diff
@@ -745,15 +745,15 @@ components:
           example: 4.0
         start:
           type: string
-    NoteFree2Read:
-      type: object
-      properties:
-        start:
-          type: string
-          format: date
-        end:
-          type: string
-          format: date
+#     NoteFree2Read:
+#       type: object
+#       properties:
+#         start:
+#           type: string
+#           format: date
+#         end:
+#           type: string
+#           format: date
```

Comment:
Line 569 also references free2read - should that similarly be commented out?
```
        free2read:
          $ref: '#/components/schemas/NoteFree2Read'
```

Reply:
yeah, you're right.
I've removed them all.